styles/globals.css

```diff
@@ -1,4 +1,8 @@
 @import url('/fonts/stylesheet.css');
+@import url('https://fonts.googleapis.com/css2?family=Fredoka:wght@300..700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@300...700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300...700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300..700&display=swap');
```

Comment:
In my PR #261, I think I also download some fonts there, including DM Sans, Fredoka One, and Nunito. Maybe we can figure out how to merge my code with yours later on.

Reply:
Yeah it shouldn't be too bad. We'll see based on who's code gets merged first.